README.md

```diff
@@ -68,8 +68,7 @@ Code Layout
 -----------
 
 The source code is generally arranged according to implementation language, in
-the `cpp`, `go`, `java` and `python` subdirectories.  The key subdirectories
```

Comment:
I think we should point people to the Java repo somewhere in this README, just in case.